src/main/java/org/alliancegenome/curation_api/model/entities/BioSampleGenomicInformation.java

```diff
@@ -29,6 +29,11 @@
 @Schema(name = "BioSampleGenomicInformation", description = "POJO that represents the BioSampleGenomicInformation")
 @AGRCurationSchemaVersion(min = "2.7.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { AuditedObject.class })
 @Indexed
+@Table(indexes = {
+	@Index(name = "biosamplegenomicinfo_bioSampleAllele", columnList = "bioSampleAllele_id"),
```

Comment:
Can these have the _index suffix for the sake of naming consistency.

Reply:
If you look at the other examples in the the model files you will see the naming assuming its not too long is `table_fieldName_index`

Reply:
Thank you for catching this. Will change this